trigger-jenkins-pipeline/README.md

```diff
@@ -0,0 +1,90 @@
+# Trigger Jenkins
+
+The following task can be used to trigger a Jenkins pipeline using CURL request from a Tekton Task.
+
+More details on Remote Access API can be found [here](https://www.jenkins.io/doc/book/using/remote-access-api/)
+
+## Install the Task
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/trigger-jenkins-pipeline/trigger-jenkins-pipeline.yaml
+```
+
+## Parameters
+
+- **JENKINS_HOST_URL**: The URL on which Jenkins is running (**Required**)
+- **JOB_NAME**: The Job name which needs to be triggered (**Required**)
+- **JENKINS_SECRETS**: The name of the secret containing the username and API token for authenticating the Jenkins (_Default_: jenkins-credentials) (**Required**)
+- **ARGS**: Extra parameters which needs to be appended in the `CURL` request. (_Default_: ""). `ARGS` is of type `array` so multiple arguments can be appended. `ARGS` can be provided as follows:-
+
+  ```yaml
+  params:
+    - name: ARGS
+      value: |
```

Comment:
Providing param values in this format is quite difficult. Is it possible for the task to generate this json in the task body based on an array of key-values?

```
params:
  - name: ARGS
    value:
        - id=123
        - appName=foo
```